wallet/src/wallet.rs

```diff
@@ -0,0 +1,185 @@
+//! # Wallet related structs
```

Comment:
I guess this structs will be replaced by the handlers in `wallet/src/actors/app/handlers/`?

Reply:
Probably, I have kept them here for the meantime but when implementing the message for the handler we will probably move each one into the appropriate handler.